index.html

```diff
@@ -2,4 +2,5 @@
 
 <h1>Hello from forloopZambia</h1>
 <p>We are learning git and github </p>
+  Thomas Munguya
```

Comment:
Html should never container naked text, wrap the text into some kind of tags, if you don't want it to have any effect then use a <span>